### PR TITLE
Add Pluracle contract

### PR DESCRIPTION
**Pull request by AugustoL**

### CODE REVIEW

File: contracts/MediumPriceOracle.sol [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "./OracleCasting.sol";
+import "./interfaces/ISignedOracle.sol";
+
+
+/**
+ * @title UintPluracle
+ * @dev An oracle of uint256 types oracles that provides a medium price.
+ */
+contract UintPluracle is Ownable {
```
Comment (jpkcambridge) [inline contracts/MediumPriceOracle.sol]: Should we make this consistent with the title of the file?

Comment (AugustoL) [reply contracts/MediumPriceOracle.sol]: yes, changing it !

File: contracts/UintPluracle.sol [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+contract UintPluracle is Ownable {
+  using SafeMath for uint256;
+
+  ISignedOracle[] oracles;
+  uint256 public _data;
+  uint256 public _reward;
+  uint256 public _lastTimestamp;
+
+  function UintPluracle(
+    uint256 reward, string dataType
```
Comment (jpkcambridge) [inline contracts/UintPluracle.sol]: Is `dataType` getting used?

File: contracts/UintPluracle.sol [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  }
+
+  function update(bytes32 data, uint256 dataTimestamp, bytes signature) public {
+    // Check that time has passed since last update
+    require(now > _lastTimestamp);
+
+    // Get all prices
+    uint256 totalPrice;
+    uint256 totalOracles;
+    for(uint8 i = 0; i < oracles.length; i ++) {
```
Comment (jpkcambridge) [inline contracts/UintPluracle.sol:37]: Should we make this uint256?

**Pull request merged**